jersey-servers/src/main/java/com/palantir/remoting1/servers/jersey/TraceEnrichingFilter.java

```diff
@@ -50,13 +51,14 @@ public void filter(ContainerRequestContext requestContext) throws IOException {
         if (Strings.isNullOrEmpty(traceId)) {
             // HTTP request did not indicate a trace; initialize trace state and create a span.
             Tracer.initTrace(Optional.<Boolean>absent(), Tracers.randomId());
-            Tracer.startSpan(operation);
+            Tracer.startSpan(operation, SpanType.SERVER_INCOMING);
```

Comment:
I think we want to capture the component (e.g. `catalog`) in addition to the operation (`GET /foo`) when starting a span.